app/modules/entities/components/layouts/author.svelte

```diff
@@ -72,39 +72,41 @@
         {/await}
       </div>
     </div>
-    <MissingEntitiesMenu
-      waiting={waitingForSubEntities}
-      questionText="A series or a work by this author is missing in the common database?"
-      {createButtons}
-    />
-    <div class="relatives-lists">
-      <RelativeEntitiesList
-        {entity}
-        claims={entity.claims['wdt:P737']}
-        label={i18n('authors_or_works_influencing_author', { name: entity.label })}
+    {#if sections}
```

Comment:
[non-blocking]
Using an await seems to better communicate why there is this block
```suggestion
    {#await waitingForSubEntities then}
```
That could still deserve a comment though, as there is no clear dependency between `sections` or `waitingForSubEntities` and what's in the block.

Reply:
besides this line has been introduced by this PR (but by f505be9ae6) i like how `if sections` documents the fact that `WorksBrowserSection` must have defined `sections` to be displayed, not only subEntities but actually sections. The `waitingForSubEntities` feels more detach from what it actually does, since future maintainer would actually need to understand first that sections is filled with subEntities. `If sections` feels more simple, saying: no sections, no cookies, nothing to display.

Actually im not even sure this `if` or `await` is even necessary, removing it does seem to hurt the display. I removed it in 11b0c9561

Reply:
done in 91d01f25c